wye/templates/base.html

```diff
@@ -139,6 +139,7 @@
         <script src="{% static 'js/libs/jquery-2.1.4.min.js' %}"></script>
         <script src="{% static 'js/libs/bootstrap.min.js' %}"></script>
         <script src="{% static 'lead_create_form.js' %}"></script>
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.15.0/jquery.validate.js"></script>
```

Comment:
Can you add jquery.validate in js library as with do with other.(Refer base.html line 139-142) apart from this it looks fine.